### PR TITLE
Move tasks in taskbar

### PR DESCRIPTION
**Pull request by RemiDufloer**
# add the possibility to move the tasks in the taskbar

With @nrainhart implement the possibility to move a task on the task bar, we add a shortcut : ` CTRL + SHIFT + ALT + ArrowLef | ArrowRight  ` 

image

image

### CODE REVIEW

**Comment by MarcusDenker** [comment]
Failing tests:

- TaskbarMorphTest (multiple)
- NoUnusedVariablesLeftTest: testNoUnusedTemporaryVariablesLeft
- ProtocolConventionsTest: testProperTestProtocolIsUsed

File: src/Morphic-Widgets-Taskbar-Tests/TaskbarMorphTest.class.st [hunk 1]
```diff
@@ -69,6 +81,71 @@ TaskbarMorphTest >> tasks [
...
+	taskbar := self.
+	world := self.
+	taskbar updateTasks.
+	taskbarMorph := TaskbarMorph new.
+	firstTask := self addTaskWithWindow.
+	secondTask := self addTaskWithWindow.
+	taskbarMorph orderedTasks: { firstTask. secondTask } asOrderedCollection.
+	
+	
+	self assert: (taskbarMorph canMoveLeft: (firstTask   morph) ) equals: false.
```
Comment (PalumboN) [inline src/Morphic-Widgets-Taskbar-Tests/TaskbarMorphTest.class.st]: You can use `self deny: ...boolean...` to expect false as result.

File: src/Zinc-HTTP-Examples/ZnStaticFileServerDelegateTest.class.st [hunk 2]
```diff
@@ -7,6 +7,15 @@ Class {
...
 
+{ #category : #private }
+ZnStaticFileServerDelegateTest >> indexHtml [
+	^ '<html>
+<head><title>Index</title></head>
+<body><h1>Hi, i''m the index page</h1></body>
+</html>
+'
+]
+
```
Comment (PalumboN) [inline src/Zinc-HTTP-Examples/ZnStaticFileServerDelegateTest.class.st]: What about the Zinc code? Seems to be not related to the PR

**Review by PalumboN — APPROVED** [review]

File: src/Zinc-HTTP-Examples/ZnStaticFileServerDelegateTest.class.st (hunk 2 above)
Comment (nrainhart) [reply src/Zinc-HTTP-Examples/ZnStaticFileServerDelegateTest.class.st]: That's because we'd started off from the `Pharo10` branch, instead of `Pharo11`. I rebased this brach over `Pharo11` to get rid of these files

**Comment by tesonep** [comment]
Thanks a lot @RemiDufloer @nrainhart !!!!
